packages/cli/src/lib/cmds/build.ts

```diff
@@ -137,5 +139,31 @@ export async function build(
   );
 
   log.info(`Signed Android App generated at "${outputFile}"`);
+
+  try {
+    const digitalAssetLinksFile = './assetlinks.json';
```

Comment:
Is this going to be relative to the directory the tool was run from? Is that what we want?

Reply:
I noticed that, but for now I'm following what we're doing for `./app-release-signed.apk`. I do think a flag to output somewhere different in the future will be useful (will be a larger change than this, as it'll touch other steps).